config/initializers/rack_rewrite.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# URL for Nuts Server deployed on Heroku, redirects download/update requests to latest release
+# on GitHub Releases with platform parameters
+RELEASE_SERVER_URL = "http://classroom-assistant-releases.herokuapp.com"
+
+Rails.application.config.middleware.insert_before(Rack::Runtime, Rack::Rewrite) do
```

Comment:
So this allows routes to reroute to an external source? Cool!

Reply:
Yea! it's so we can reroute requests to the release server Heroku app. We can also switch out the release server in the future if we want.

The server is running at `http://classroom-desktop-releases.herokuapp.com` since it hasn't been renamed yet.